common/src/MappedFile.C

```diff
@@ -168,17 +177,7 @@ void MappedFile::closeMappedFile(MappedFile *&mf)
 
```

Comment:
decrement and then test of `refCount` is not thread safe.  `refCount` needs to be atomic and then and then atomically decrement and return the pre decrement value; and then test the value.